utilslib/dr.py

```diff
@@ -91,10 +93,13 @@ def parse_key(key):
         tuble containing the fields in the key based on '/' seperator.
         """
         fields = key.split('/')
-        if len(fields) != 4:
+        if len(fields) == 4:
+            return fields[0], fields[1], fields[2], fields[3]
+        elif len(fields) == 5:
+            return fields[0], fields[1], fields[2], fields[4]
+        else:
             raise Exception(
                 "key should comprise /cluster/namespace/kind/name")
```

Comment:
no leading / and version (with / changed to _) after kind

Reply:
True, i've not cleard this up yet but will do.